### ISSUE TITLE
Occasional errors running workflow

### ISSUE THREAD

**Issue by rodluger** [rodluger/showyourwork#57]
This error has been popping up occasionally:

```
.showyourwork/scripts.json
WorkflowError:
Rule parameter depends on checkpoint but checkpoint output is not defined as input file for the rule. Please add the output of the respective checkpoint to the rule inputs.
```

Usually re-running the workflow fixes it. This should be an easy fix, though -- let's look into it.

**Comment by rodluger** [comment]
@katiebreivik Fixed in `v0.1.32`.

**Comment by rodluger** [comment]
@katiebreivik That release was buggy... Fixed for real in `v0.1.33`
